src/Interval.h

```diff
@@ -53,6 +53,24 @@ class Interval : public Range
   std::string json () const;
   std::string dump () const;
 
+  bool operator== (const Interval& other) const
```

Comment:
analog to `Range::operator==` these should go into `Interval.cpp`

Reply:
Done